iib/workers/tasks/build.py

```diff
@@ -119,9 +119,16 @@ def _build_image(dockerfile_dir: str, dockerfile_name: str, request_id: int, arc
         exc_msg=f'Failed to build the container image on the arch {arch}',
     )
     log.debug('Verifying that %s was built with expected arch %s', destination, arch)
-    archmap = get_worker_config()['iib_supported_archs']
+    archmap = worker_config['iib_supported_archs']
     destination_arch = get_image_label(local_destination, 'architecture')
 
+    if not destination_arch:
+        log.warn(
+            '"architecture" label not found under "Labels".'
+            'Skipping the check that confirms if the architecture label was set correctly.'
```

Comment:
Can the warning be rewritten, so the sentence would start with a capital letter? I find logs more readable that way (looking for capital letters, when looking for another log message).

Reply:
Done. :)